### ISSUE TITLE
Consider starting a public discussion channel (e.g. Discord, Gitter) to connect with external contributors

### ISSUE THREAD

**Issue by luyangkenneth** [MarkBind/markbind#815]

**Comment by yamgent** [comment]
`se-edu` currently uses Gitter (which is also used by other open-source projects such as Angular), so I think Gitter would be very suitable for this purpose.

**Comment by tlylt** [comment]
I think discord is a popular option that we can explore for both public and private discussions, especially given that currently for internal conversations, Slack's free tier has the "Messages and files older than 90 days are hidden" limitation.

WDYT @MarkBind/admins ?

**Comment by kaixin-hc** [comment]
Discord isn't truly "open" - moving things to an alternative platform might feel like more hoops to jump through before contributing. I do like Discord though.

Personally I am against a public discussion channel (besides ofc this issue tracking) until the need becomes clearer - we would need to make sure it is maintained and checked and active, which may be difficult

**Comment by tlylt** [comment]
A public instant messaging channel/forum that can keep the chat history is something that I would want...so that the context around quick discussions/announcements can be tracked, referenced and communicated to anyone who's interested. I know we do use telegram now for CS3281/82, maybe a public telegram group might work as well?

**Comment by kaixin-hc** [comment]
It's true most contributors are on telegram... but telegram sucks for anonymous communication, which I think would be a big deterrent.

Would also support moving away from Slack for internal communications haha. Out of these options Discord is probably the best, but I still think the context should be archived, tracked, referenced and communicated within the issue tracker.

**Comment by tlylt** [comment]
> but I still think the context should be archived, tracked, referenced and communicated within the issue tracker.

There's still value in providing a chat/channel for real-time discussion. If you recall the markbind archive task...issues can have hundreds of comments and get really hard to follow:) This alternative would probably help a little for that kind of scenario
